docs/tutorials/index.md

```diff
@@ -1,42 +1,41 @@
-# Tutorials
+# Simulation tutorials
 
-This page explains how to try out Autoware through simulations.
-
-There are some methods and types of simulations.
-Read the following sections and choose a simulation method/type you want
-
-See the left sidebar for how to run each simulation.
-
-!!! warning
-
-    The namings of each simulation are under discussion and would be changed in the near future.
+Simulations provide a way of verifying Autoware's functionality before field testing with an actual vehicle.  
+There are three main types of simulation that can be run ad hoc or via a scenario runner.
 
 ## Simulation methods
 
 ### Ad hoc simulation
 
-Ad hoc simulation is a simulation method that you run simulations flexibly on your local machine.
-It is recommended for your first trial.
+Ad hoc simulation is a flexible method for running basic simulations on your local machine, and is the recommended method for anyone new to Autoware.
 
 ### Scenario simulation
 
-Scenario simulation is a simulation method that a scenario runner runs simulations based on scenarios you defined.
-It is often run automatically on a CI system, but you can also run this simulation on your local machine.
+Scenario simulation uses a scenario runner to run more complex simulations based on predefined scenarios.
+It is often run automatically for continuous integration purposes, but can also be run on a local machine.
 
 ## Simulation types
 
-Autoware supports three types of simulations.  
-Note that it is just a rough classification, and that there are several minor derivative versions.
-
 ### Planning simulation
 
-Planning simulation is a simulation type that uses simple dummy data and tests mainly planning/control components.
+Planning simulation uses simple dummy data to test the Planning and Control components - specifically path generation, path following and obstacle avoidance. It verifies that a vehicle can reach a goal destination while avoiding pedestrians and surrounding cars, and is another method for verifying the validity of Lanelet2 maps.
+
+#### How does planning simulation work?
+
+1. Generate a path to the goal destination
+2. Control the car along the generated path
+3. Detect and avoid any humans or other vehicles on the way to the goal destination
+
+!!! note
+
+    For real-world operation, we also need to detect and handle traffic lights.
+    However, planning simulation does not verify this functionality.
```

Comment:
Actually, it's not true. `scenario_simulator_v2` supports traffic lights.
@WJaworskiRobotec Could you give us your feedback on the descriptions for this section? :pray:

Reply:
In general looks fine. I have 2 more comments:
1. I thought that scenario_simulation is one way of running of the planning simulation but I can see that these two are separated in the description. But I might be wrong, then it's totally fine
2. There is additional way of testing : random_test_runner that generated random traffic in the experiment. This one should be described here as well

Reply:
@kenji-miyake @WJaworskiRobotec So, it is possible to set the state of a traffic light on the map in the Planning Simulator and then test how the ego vehicle responds to the traffic light state, and this can be done for both ad hoc testing and using the scenario simulator? Well I learned something today.

I will remove that note then.

Reply:
@WJaworskiRobotec Since the random test runner is part of scenario simulation, would it be okay to describe that within the scenario simulation sub-section? If so, I think that change should be submitted as a separate PR in order to keep changes small and thus make it easier for reviews to check.

Reply:
@LalithVipulananthan sure, I fully agree that it can be done in the separate PR.

Reply:
> and this can be done for both ad hoc testing and using the scenario simulator?

It's supported only with the scenario simulator, and not available with ad hoc testing.

Reply:
@kenji-miyake How about I change the note to say the following instead?

`Traffic light handling functionality can also be tested with the Planning Simulator, but only via scenario simulation.`

Reply:
Looks good! Thanks. :pray:

Reply:
@kenji-miyake Based on #129 , it looks as if traffic light handling can be tested with ad hoc planning simulations as well?

Reply:
@LalithVipulananthan It seems so, but I'm not familiar with the feature.
@soblin @shmpwk @kminoda Could you please give some feedback on the explanation of this section? :pray:

Reply:
@soblin @shmpwk @kminoda Hi guys. I just wanted to ping you about Miyake-san's request above. Once this last question has been resolved, this PR can be merged and work can continue with #126, #127 and #129.

Reply:
Yes, I think traffic light handling can be tested with ad-hoc simulation (as in https://github.com/autowarefoundation/autoware-documentation/pull/129). So I guess the following note is not necessary and can be removed.
```
!!! note

    Traffic light handling functionality can also be tested with the Planning Simulator, but only via scenario simulation.
```